### ISSUE TITLE
Hundreds of rabbits

### ISSUE THREAD

**Issue by Braffolk** [capnkirok/animaniamod#234]
### Expected Behavior
Expected to have a normal amount of rabbits.

### Actual Behavior
Got hundreds of rabbits, but most of them oddly only spawn on this type of mountain (technically it's the Minecraft Desert biome). After I've killed most of them, new ones aren't really popping out of nowhere. so that can't be the issue, though slowly more of them seem to appear. The server has been running for around 2-3 hours with the 1.6.2 animania update. Don't think we had issues like this before.

### Version of Minecraft, version of Animania, Single Player or Server
Server, this area should be in spawn chunks. Animania 1.6.2, Minecraft 1.12.2

### Screenshots encouraged
![](https://i.imgur.com/vSsiO7G.png)

**Comment by Tschipp** [comment]
Oh wow... that's a lot of rabbits. It's impossible that they are breeding though, as they would have hit the limit a long time ago (and they need food and water to breed, none of which is around here), so they must be spawning

**Comment by RayHell** [comment]
I have the same issue with horses right now

**Comment by glepet1962** [comment]
I am also having this issue with horses.

**Comment by capnkirok** [comment]
@glepet1962 which biome type is this happening?

**Comment by Braffolk** [comment]
Update: I checked the configs on my server, B:replaceVanillaRabbits was set to true, all other replace settings were set to false (no other animania animal was mass-spawning).

I have now set it to false and thus far the issue seems to have disappeared. In case it reappears, I will post an update, but so far rabbits seem to be spawning normally.

**Comment by glepet1962** [comment]
> @glepet1962 which biome type is this happening?

The plains.

**Comment by T0sha** [comment]
![2018-12-05_18 58 51](https://user-images.githubusercontent.com/45928354/50064707-d058f800-016f-11e9-94be-06f4cfe0adf2.png)
 
Having the same issue in the desert. This was after we went and killed a load of them. I will try and locate the file that Braffolk is talking about and update it to see if it fixes the issue.  Thanks.

**Comment by sekelsta** [comment]
Just going to guess here, but does this happen only on worlds that used to be vanilla and had vanilla animals get converted to Animania ones, and do you get more rabbits every time you load the world/chunk? If so, it might be because in the onEntitySpawn() function the entities are prevented from spawning but not removed from the save file. If that's the case, it can be solved by adding event.getEntity.setDead() instead of just event.setCancelled().
Edit: nevermind, that isn't what's happening

**Comment by BigKid1973** [comment]
I've got the same issues on my dedicated server. It started with sheeps. It gets's worse the more sheeps you've got... it's like they are breeding (I do not say they are)... I had to slaughter masses of them to keep it at bay. Once we started breeding other animals it started to happen with them too. So it doesn't seem to normal spawning but somehow related to animals that are already there... Other strange thing I noticed are animals seemingly going through fences (I've seperated the different animals with fences) ... Probably the problem is getting worse with every restart of the server but I'm not sure. After we had the server running with some chunks loaded over night and restarted (unfortunately without loggin in before - so I do not know if the problem was there before the restart) we only (or at least almost) had male animals left with most of our animals and no (or almost no) females... before we logged out we had just a few males and mainly females for all animals... Really Strange...
